Quote TKEXECUTABLE for Mac OS X 10.5 and earlier where the path contains spaces
(/System/Library/Frameworks/Tk.framework/Resources/Wish Shell.app/Contents/MacOS/Wish Shell)
--- a/git-gui/GIT-GUI-BUILD-OPTIONS.in.orig	2025-06-16 00:11:33.000000000 -0500
+++ b/git-gui/GIT-GUI-BUILD-OPTIONS.in	2025-07-27 15:44:43.000000000 -0500
@@ -4,4 +4,4 @@
 SHELL_PATH=@SHELL_PATH@
 TCLTK_PATH=@TCLTK_PATH@
 TCL_PATH=@TCL_PATH@
-TKEXECUTABLE=@TKEXECUTABLE@
+TKEXECUTABLE="@TKEXECUTABLE@"
